mozilla_django_oidc_db/backends.py

```diff
@@ -152,56 +125,29 @@ def authenticate(  # type: ignore
         if request is None:
             return None
 
-        # Load the config to apply and check if this backend should be considered to
-        # authenticate the user.
-        self.config_class = lookup_config(request)
+        self.request = request
+        # Check if this backend should be considered to authenticate the user.
         is_candidate = self._check_candidate_backend()
         if not is_candidate:
             return None
 
         # Allright, now try to actually authenticate the user.
         user = super().authenticate(request, nonce=nonce, code_verifier=code_verifier)
 
-        # Store the config class name on the user, so that we can store this in the user's
+        # Store the config class identifier on the user, so that we can store this in the user's
         # session after they have been successfully authenticated (by listening to the `user_logged_in` signal)
         if user:
-            options = self.config_class._meta
-            user._oidcdb_config_class = f"{options.app_label}.{options.object_name}"  # type: ignore
+            user._oidcdb_config_identifier = self._config.identifier
```

Comment:
please add the `# type: ignore` back because this attribute is unknown and static type checkers will trip over this :)

I think one of the next courses of action in this package is adding pyright type checking to CI too because we keep going back-and-forth with type annotations too.